Applications/WebPortal/WebApp/library/source/pages/dashboard_page/http_dashboard_model.ts

```diff
@@ -60,6 +60,13 @@ export class HttpDashboardModel extends DashboardModel {
     return model;
   }
 
+  public makeGroupModel(group: Beam.DirectoryEntry): HttpGroupModel {
+    if(!this.groupModel || group.id !== this.groupModel.group.id) {
```

Comment:
Look at ```makeAccountModel```, the way to implement this method is to see if a model already exists for the ```group``` and if it does then return it. If no existing model exists for the ```group```, only then do you construct a new model for the ```group``` and you save it for future reference.